plotter/heat.go

```diff
@@ -93,7 +98,7 @@ func NewHeatMap(g GridXYZ, p palette.Palette) *HeatMap {
 
 // Plot implements the Plot method of the plot.Plotter interface.
 func (h *HeatMap) Plot(c draw.Canvas, plt *plot.Plot) {
-	if h.Min >= h.Max {
+	if h.Min > h.Max {
 		panic("heatmap: non-positive Z range")
```

Comment:
The panic message should say negative, the condition and the changes in this PR allow empty Z range. I'll then approve.